app/javascript/pages/account/create-account.tsx

```diff
@@ -176,6 +180,8 @@ const CreateAccountContent = ({ register, watch, errors }: SectionProps) => {
   )
 }
 
+const fieldOrder = [...nameSortOrder, ...dobSortOrder, ...emailSortOrder, ...passwordSortOrder]
```

Comment:
thought: We should try to centralize all the field name strings, and also have a centralized sort order of those strings. We would have less export statements that way, although we may have more import statements.